README.md

```diff
@@ -55,3 +58,6 @@ The SSH tunnel can then be used for things like connecting to an RDS database th
 
 `aws-connect -s -t CLUSTER=prod`
 
+5. Run SSM Document named shell-script on shopify with default profile and parameters 'param1 param 2'. Github token and document are required: 
```

Comment:
change parameters to arguments.  As mentioned above, let's make github token optional if the scripts are in a public repo